packages/@aws-cdk/aws-appmesh/lib/virtual-gateway.ts

```diff
@@ -187,6 +188,7 @@ export class VirtualGateway extends VirtualGatewayBase {
     }
 
     const accessLogging = props.accessLog?.bind(this);
+    const backendDefaultsTlsClientPolicy = props.backendDefaults?.tlsClientPolicy;
```

Comment:
Since this variable is used exactly once, I don't think it's very valuable. I would inline it.

Reply:
Inlined `props.backendDefaults?.tlsClientPolicy`.